miasm2/ir/ir.py

```diff
@@ -52,6 +52,17 @@ def get_dst(self):
         self._dst = dst
         return dst
 
+    def set_dst(self, value):
```

Comment:
Because this is a public method (not an internal one), can you specify its behaviour in a (again tiny) docstring ?